packages/apps/human-app/server/src/main.ts

```diff
@@ -1,8 +1,27 @@
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
+import { ConfigService } from '@nestjs/config';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
-  await app.listen(3000);
+
+  const configService: ConfigService = app.get(ConfigService);
+
+  const config = new DocumentBuilder()
+    .addBearerAuth()
+    .setTitle('Human APP API')
+    .setDescription('Swagger Human APP API')
+    .setVersion('1.0')
+    .build();
+  const document = SwaggerModule.createDocument(app, config);
+  SwaggerModule.setup('swagger', app, document);
+
+  const host = configService.get<string>('HOST', 'localhost');
+  const port = configService.get<string>('PORT', '5010');
+
+  await app.listen(port, host, async () => {
+    console.info(`Human APP server is running on http://${host}:${port}`);
```

Comment:
maybe we will use logger instead of console log

Reply:
Fixed in commit 5caea62b